test/unit/eval_algorithms/test_qa_accuracy_semantic_robustness.py

```diff
@@ -105,11 +107,13 @@ class TestCaseQAAccuracySemanticRobustnessEvaluateSample(NamedTuple):
                     EvalScore(name=QUASI_EXACT_MATCH_SCORE, value=1.0),
                     EvalScore(name=PRECISION_OVER_WORDS, value=1.0),
                     EvalScore(name=RECALL_OVER_WORDS, value=1.0),
+                    EvalScore(name=BERT_SCORE, value=0.48504769802093506),
```

Comment:
We should be mocking the bertscore model in the unit tests. We need the unit tests to validate that all of the new logic that was added to the transform pipeline is correct. Currently, none of this logic is getting tested.

Reply:
Ok, will work on that